client/my-sites/domains/email-providers-upsell/index.jsx

```diff
@@ -10,47 +10,52 @@ import { useTranslate } from 'i18n-calypso';
  * Internal dependencies
  */
 import { domainAddNew } from 'calypso/my-sites/domains/paths';
+import CalypsoShoppingCartProvider from 'calypso/my-sites/checkout/calypso-shopping-cart-provider';
+import EmailProvidersComparison from 'calypso/my-sites/email/email-providers-comparison';
 import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
-import HeaderCake from 'calypso/components/header-cake';
-import PromoCard from 'calypso/components/promo-section/promo-card';
-
-/**
- * Style dependencies
- */
-import emailIllustration from 'calypso/assets/images/email-providers/email-illustration.svg';
 
 export default function EmailProvidersUpsell( { domain } ) {
 	const selectedSiteSlug = useSelector( getSelectedSiteSlug );
 	const translate = useTranslate();
 
-	const handleGoBack = () => {
-		page( domainAddNew( selectedSiteSlug ) );
-	};
-
-	const image = {
-		path: emailIllustration,
-		align: 'right',
-	};
+	const comment = '%(domainName)s is the domain name, e.g example.com';
 
 	return (
 		<>
-			<HeaderCake onClick={ handleGoBack }>
-				{ translate( 'Register %(domain)s', { args: { domain } } ) }
-			</HeaderCake>
-
-			<PromoCard
-				isPrimary
-				title={ translate( 'Add a professional email address to %(domainName)s', {
-					args: {
-						domainName: domain,
-					},
-					comment: '%(domainName)s is the domain name, e.g example.com',
-				} ) }
-				image={ image }
-				className="email-providers-upsell__action-panel"
-			>
-				<p>{ translate( 'No setup or software required. Easy to manage from your dashboard.' ) }</p>
-			</PromoCard>
+			<CalypsoShoppingCartProvider>
+				<EmailProvidersComparison
+					backPath={ domainAddNew( selectedSiteSlug ) }
+					headerTitle={ ( domainName ) =>
+						translate( 'Register %(domainName)s', { args: { domainName }, comment } )
+					}
+					isEmailHeaderShown={ false }
+					isEmailForwardingCardShown={ false }
+					isSkippable
+					onSkipClick={ () => {
+						page( `/checkout/${ selectedSiteSlug }` );
+					} }
+					promoHeaderTitle={ ( domainName ) =>
+						translate( 'Add a professional email address to %(domainName)s', {
+							args: {
+								domainName,
+							},
+							comment,
+						} )
+					}
+					promoHeaderDescription={ ( domainName ) =>
+						translate(
+							'Pick from one of our flexible options to connect your domain with email and start getting emails @%(domainName)s today.',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **13** times:
`translate( 'Pick one of our flexible options to connect your domain with email and start getting emails @%(domainName)s today.' )` *ES Score: 12*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/pull/55183)